src/program.js

```diff
@@ -174,13 +185,21 @@ export function defaultVersionGetter(
   }
 }
 
+// TODO: add pipes to Flow type after https://github.com/facebook/flow/issues/2405 is fixed
+
+type MainParams = {
+  getVersion: Function,
+  commands: Object,
+  argv: Array<any>,
+  runOptions: Object,
```

Comment:
this should be optional: `runOptions?:` since it has a default value in the function signature